videosink/display.go

```diff
@@ -25,11 +25,14 @@ import (
 	"image/png"
 	"net/http"
 	"sync"
+	"time"
 
 	"periph.io/x/conn/v3/display"
 )
 
 const defaultJPEGQuality = 95
+const defaultKeepAliveInterval = time.Minute
+const defaultMinFrameInterval = time.Second / 15
```

Comment:
You may want `(time.Second + time.Second/30) / 15`

Reply:
What would be the practical benefit? Per https://go.dev/play/p/aadQgiosT-z your proposal and my code differ by ~2.2ms—not relevant.